components/Task/Task.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+const StyledTask = styled.li`
+  border: 1px solid black;
+  padding: 1rem;
+  margin: 0.5rem;
+`;
+
+const StyledTaskInfo = styled.div`
```

Comment:
maybe a semantic element like section would be more fitting than div? (same for StyledPriority)